dotCMS/src/main/java/com/dotmarketing/portlets/templates/design/bean/TemplateLayout.java

```diff
@@ -148,4 +150,30 @@ public void setContainers ( final List<ContainerUUID> containers, final Boolean
 
     }
 
+    public List<String> getContainersId() {
+        final List<String> containerIdentifiers = new ArrayList<>();
+        final List<TemplateLayoutRow> rows = this.body.getRows();
+
+        for (final TemplateLayoutRow row : rows) {
+            final List<TemplateLayoutColumn> columns = row.getColumns();
+
+            for (final TemplateLayoutColumn column : columns) {
+                final List<ContainerUUID> columnContainers = column.getContainers();
+                containerIdentifiers.addAll(
+                        columnContainers.stream()
+                                .map((ContainerUUID containerUUID) -> containerUUID.getIdentifier())
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)